.circleci/config.yml

```diff
@@ -1,5 +1,181 @@
 version: 2.1
 
+orbs:
+  # Using inline orb for now
+  getting-started-smoke-test:
+    orbs:
+      macos: circleci/macos@2
+      android: circleci/android@2.0.3
+      flutter-orb: circleci/flutter@1.0.1
```

Comment:
Latest version is `1.1.0` from what I can see. Any reason to not use that?

Reply:
Yes we can update to latest version